src/handlers/issue/generate-erc721-permit-signature.ts

```diff
@@ -1,27 +1,27 @@
-import { BigNumber, BigNumberish, ethers, utils } from "ethers";
+import { BigNumber, ethers, utils } from "ethers";
 import { getPayoutConfigByNetworkId } from "../../helpers/payout";
 import { MaxUint256 } from "@uniswap/permit2-sdk";
 import { keccak256, toUtf8Bytes } from "ethers/lib/utils";
 
 const NFT_MINTER_PRIVATE_KEY = process.env.NFT_MINTER_PRIVATE_KEY as string;
-const NFT_CONTRACT_ADDRESS = process.env.NFT_CONTRACT_ADDRESS as string;
-const NFT_CONTRACT_CHAIN_ID = parseInt(process.env.NFT_CONTRACT_CHAIN_ID as string);
+const NFT_CONTRACT_ADDRESS = "0x6a87f05a74AB2EC25D1Eea0a3Cd24C3A2eCfF3E0";
+const NFT_CONTRACT_NETWORK_ID = 100;
```

Comment:
I think we should deploy on all the networks that `permit2` is on and then take this value from the config since we already have it there! For now, @whilefoo, please take it from the existing value in the config.